torchmetrics/functional/image/ms_ssim.py

```diff
@@ -0,0 +1,179 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Optional, Sequence, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+from typing_extensions import Literal
+
+from torchmetrics.functional.image.ssim import _ssim_compute, _ssim_update
+
+
+def _get_normalized_sim_and_cs(
+    preds: Tensor,
+    target: Tensor,
+    kernel_size: Sequence[int] = (11, 11),
+    sigma: Sequence[float] = (1.5, 1.5),
+    reduction: str = "elementwise_mean",
+    data_range: Optional[float] = None,
+    k1: float = 0.01,
+    k2: float = 0.03,
+    normalize: Optional[Literal["relu", "simple"]] = None,
+) -> Tuple[Tensor, Tensor]:
+    sim, contrast_sensitivity = _ssim_compute(
+        preds, target, kernel_size, sigma, reduction, data_range, k1, k2, return_contrast_sensitivity=True
+    )
+    if normalize:
+        sim = torch.relu(sim)
+        contrast_sensitivity = torch.relu(contrast_sensitivity)
+    return sim, contrast_sensitivity
+
+
+def _ms_ssim_compute(
+    preds: Tensor,
+    target: Tensor,
+    kernel_size: Sequence[int] = (11, 11),
+    sigma: Sequence[float] = (1.5, 1.5),
+    reduction: str = "elementwise_mean",
+    data_range: Optional[float] = None,
+    k1: float = 0.01,
+    k2: float = 0.03,
```

Comment:
i wonder if we should just remove k1 and k2 as they are just used for stabilizing the division in ssim

Reply:
@SkafteNicki Do you have any opinion on if we should keep `k1` and `k2`? [PyTorch implementation](https://github.com/jorge-pessoa/pytorch-msssim/blob/master/pytorch_msssim/__init__.py) does not support these arguments, while the original [TF implementation](https://www.tensorflow.org/api_docs/python/tf/image/ssim_multiscale) has these two arguments.

Reply:
lets just keep it as it is :]